supervision/dataset/core.py

```diff
@@ -197,7 +197,10 @@ def as_pascal_voc(
 
     @classmethod
     def from_pascal_voc(
-        cls, images_directory_path: str, annotations_directory_path: str
+        cls,
```

Comment:
There is an unused import at the top of this file.

`from supervision.utils.file import list_files_with_extensions`

Please remove it if you can.